tests/profiles/googlefonts_test.py

```diff
@@ -2283,13 +2283,7 @@ def test_check_italic_angle():
 
     # italic-angle, style, fail_message
     test_cases = [
-        [1, "Italic", FAIL, "positive"],
-        [0, "Regular", PASS, None], # This must PASS as it is a non-italic
-        [-21, "ThinItalic", WARN, "over-minus20-degrees"],
-        [-30, "ThinItalic", WARN, "over-minus20-degrees"],
         [-31, "ThinItalic", FAIL, "over-minus30-degrees"],
-        [0, "Italic", FAIL, "zero-italic"],
-        [-1,"ExtraBold", FAIL, "non-zero-normal"]
     ]
```

Comment:
I guess all test-cases for Google Fonts should remain intact, as we expect the behavior not to have changed.

Reply:
In the spirit of data/code sparseness I thought I'd override only the one case where the GF tests would differ from the OT tests, because those tests are still complete.

Reply:
I think it is better to be thorough on code-testing. Please leave all test-cases as they were before in the google fonts profile tests